### PR TITLE
Some Sidekick quickstart edits.

### PR DESCRIPTION
**Pull request by mdwelsh**
Made some proposed edits to the Sidekick quickstart. Definitely a few things we need to chase down in the next couple of days, but all doable.

I recommend we just push these changes to the live docs site and work off of `main` since I don't think it gets so much traffic that people are gonna get confused if we start rolling things out.

### CODE REVIEW

File: packages/docs/docs/sidekicks-quickstart.md [hunk 1]
```diff
@@ -27,212 +34,193 @@ At the end of this quickstart you will:
...
 ### b) Install Node.js
 
-Install the current [LTS version of Node.js](https://nodejs.org/en).
+Sidekicks are based on AI.JSX, which in turn relies on Node.js, a JavaScript
+runtime. Install the current [LTS version of Node.js](https://nodejs.org/en).
 
 ### c) Install the Fixie CLI
 
-In your terminal execute the following:
+The Fixie command-line interface is provided by the [fixie](https://www.npmjs.com/package/fixie) package in NPM. Install it by running:
```
Comment (NickHeiner) [inline packages/docs/docs/sidekicks-quickstart.md:65]:
```suggestion
The Fixie command-line interface is provided by the [fixie](https://www.npmjs.com/package/fixie) package in npm. Install it by running:
```

npm is not an acronym.

File: packages/docs/docs/sidekicks-quickstart.md [hunk 2]
```diff
@@ -27,212 +34,193 @@ At the end of this quickstart you will:
...
+The Fixie Console page gives you a simple web interface to interact with your
+Sidekick, but you're not limited to this interface. You can chat with your
+Sidekick directly via a REST API, or embed the Sidekick chat UI in your own
+web app.
 
-We are going to test the sidekick using four different methods.
+### Method 1: Via the REST API
 
-### Method 1: Via curl
+> TODO(matt): Figure out the right invocation for this.
```
Comment (NickHeiner) [inline packages/docs/docs/sidekicks-quickstart.md:176]: See https://github.com/fixie-ai/fixie/blob/5452e48b739f9941e3facabaad9a2f1d4f59f49c/src/generic-sidekick-frontend/lib/fixie-api.ts for an example of how this API is used.

**Comment by NickHeiner** [comment]
> I recommend we just push these changes to the live docs site and work off of main

I feel pretty strongly that if we do this, we should remove the `todo`s first. But I won't gatekeep on it.

File: packages/docs/docs/sidekicks-quickstart.md [hunk 3]
```diff
@@ -8,10 +8,17 @@ import Step3Profit from '../static/img/step-3-profit.png'
...
+Sidekicks can be built and deployed in minutes. This quickstart will walk you through
+the entire process of building and deploying your own Sidekick.
+
+This tutorial relies on both AI.JSX as well as the [Fixie](https://fixie.ai) cloud
+platform, which provides a suite of APIs and tools for hosting and managing
+Sidekicks. It is possible to build and deploy Sidekicks without Fixie, but
+using Fixie makes the process much easier.
 
 :::info What You Will Do
 At the end of this quickstart you will:
```
Comment (NickHeiner) [inline packages/docs/docs/sidekicks-quickstart.md:24]: I know it's not in this diff, but `Prereq's installed` is a typo. Prereqs is not possessive.

File: packages/docs/docs/sidekicks-quickstart.md [hunk 4]
```diff
@@ -27,212 +34,193 @@ At the end of this quickstart you will:
...
+runtime. Install the current [LTS version of Node.js](https://nodejs.org/en).
 
 ### c) Install the Fixie CLI
 
-In your terminal execute the following:
+The Fixie command-line interface is provided by the [fixie](https://www.npmjs.com/package/fixie) package in NPM. Install it by running:
 
 ```terminal
-npm install -g @fixieai/fixie@latest
+$ npm install -g fixie@latest
```
Comment (NickHeiner) [inline packages/docs/docs/sidekicks-quickstart.md:68]: I think this is actually not what we want. The code block has a built-in copy button. If we add a `$` to the beginning of the line, then the user can't actually copy/paste directly into their terminal. They have to take the `$` back out.

image

File: packages/docs/docs/sidekicks-quickstart.md [hunk 5]
```diff
@@ -27,212 +34,193 @@ At the end of this quickstart you will:
...
-🥡 Revision a1a29ee1 was deployed to https://console.fixie.ai/agents/sarah/sidekick-acme
+❯ fixie deploy
+🦊 Deploying agent mdw/fixie-sidekick-template...
+🦊 Creating new agent mdw/fixie-sidekick-template...
+⠋  🚀 Deploying... (hang tight, this takes a minute or two!)
+✔ Agent fixie-sidekick-template is running at: https://console.fixie.ai/agents/mdw/fixie-sidekick-template
 ```
 
-As we make changes to our sidekick, we can simply stop serving our sidekick with `Control+C` and then serve up our new changes with the `serve` command as we did above.
+<img src={Step3Profit} alt="Step 3: Profit." width="300"/>
```
Comment (NickHeiner) [inline packages/docs/docs/sidekicks-quickstart.md:134]:
```suggestion
<img src={Step3Profit} alt="" width="300"/>
```

For a18y, `alt` should be `''` for "purely decorative" content.

File: packages/docs/docs/sidekicks-quickstart.md [hunk 6]
```diff
@@ -27,212 +34,193 @@ At the end of this quickstart you will:
...
+### Method 2: Via the Fixie Console
 
 - In your browser navigate to the [Fixie dashboard](https://console.fixie.ai/).
-- Click on your agent.
-- Enter a question for the sidekick. e.g. "What can you do?"
+- Click on your Sidekick.
+- Enter a question for the Sidekick. e.g. "What can you do?"
 
-Note: you can also access your agent directly at:
+You can also access your agent directly at:
```
Comment (NickHeiner) [inline packages/docs/docs/sidekicks-quickstart.md:203]: This section feels redundant. I don't think we need to both provide a direct link to the dev console, and also tell people how to go to the front door of the dev console and then click through it.

File: packages/docs/docs/sidekicks-quickstart.md [hunk 7]
```diff
@@ -27,212 +34,193 @@ At the end of this quickstart you will:
...
 ```terminal
 https://console.fixie.ai/agents/<your user name>/<your sidekick name>
 ```
 
 ### Method 3: Via Embedding in Another Web App
 
-TODO could we just include a simple web page in the template code that can be served up locally?
-
-_TODO Matt: need to figure out if we add this or not_
+> TODO(matt): Figure out what the right thing to do here is.
```
Comment (NickHeiner) [inline packages/docs/docs/sidekicks-quickstart.md:211]: I would punt on this for now

**Review by NickHeiner — APPROVED** [review]

File: packages/docs/docs/sidekicks-quickstart.md (hunk 3 above)
Comment (benlower) [reply packages/docs/docs/sidekicks-quickstart.md:24]: Right it's not possessive. "Prereq's" is a commonly used abbreviation of the full word.

File: packages/docs/docs/sidekicks-quickstart.md [hunk 8]
```diff
@@ -27,212 +34,193 @@ At the end of this quickstart you will:
...
 
-_TODO: Matt: right now the .env file is in the .gitignore which means they would have to create it. thoughts on the best route here? don't want to set ppl up to accidentally commit secrets._
+You can also download the code directly from [the source](https://github.com/fixie-ai/fixie-sidekick-template). While you're there, give us a star! 🦊
 
-### c) Deploy Sidekick to Fixie
+## Step 2: Build the Sidekick code
 
-The moment of truth has arrived! Let's deploy our sidekick to Fixie!
+> TODO(matt): We need to set up a public Corpus ID that the template Sidekick can
+> use out of the box without the user having to create one. I am prioritizing this!
```
Comment (benlower) [inline packages/docs/docs/sidekicks-quickstart.md:102]: thanks! which corpus?

**Review by benlower — CHANGES_REQUESTED** [review]

File: packages/docs/docs/sidekicks-quickstart.md (hunk 4 above)
Comment (benlower) [reply packages/docs/docs/sidekicks-quickstart.md:68]: incorporating this change. good point!

File: packages/docs/docs/sidekicks-quickstart.md (hunk 5 above)
Comment (benlower) [reply packages/docs/docs/sidekicks-quickstart.md:134]: done

File: packages/docs/docs/sidekicks-quickstart.md (hunk 3 above)
Comment (NickHeiner) [reply packages/docs/docs/sidekicks-quickstart.md:24]: heh, ok, I buy the abbreviation angle. :smile:

File: packages/docs/docs/sidekicks-quickstart.md [hunk 9]
```diff
@@ -27,212 +34,193 @@ At the end of this quickstart you will:
...
-_TODO Matt: this is currently broken. not sure what the plan is to fix._
-
-This should display something like this:
-
-TODO
-
-### Method 2: Via the Fixie Dashboard
+### Method 2: Via the Fixie Console
 
 - In your browser navigate to the [Fixie dashboard](https://console.fixie.ai/).
```
[the remark this answers is not in the document]
Comment (benlower) [reply packages/docs/docs/sidekicks-quickstart.md:199]: done

File: packages/docs/docs/sidekicks-quickstart.md (hunk 1 above)
Comment (NickHeiner) [reply packages/docs/docs/sidekicks-quickstart.md]: https://www.npmjs.com/package/npm?activeTab=readme :smile:

File: packages/docs/docs/sidekicks-quickstart.md (hunk 7 above)
Comment (benlower) [reply packages/docs/docs/sidekicks-quickstart.md:211]: ok removing the part about embedding for now. opened issue https://github.com/fixie-ai/ai-jsx/issues/284 to track the work for doing this in future.

File: packages/docs/docs/sidekicks-quickstart.md (hunk 8 above)
Comment (benlower) [reply packages/docs/docs/sidekicks-quickstart.md:102]: opened bug https://github.com/fixie-ai/fixie/issues/1670 to track this work.

File: packages/docs/docs/sidekicks-quickstart.md [hunk 10]
```diff
@@ -27,212 +34,193 @@ At the end of this quickstart you will:
...
-When developing or updating a Sidekick, we most likely will be making many changes and testing things as we go. In this case, we want
-to deploy things as quickly as possible.
+## Step 3: Deploy the Sidekick
 
-From the directory where you have your sidekick code, execute the following in your terminal:
+In the `fixie-sidekick-template` directory, simply run:
 
 ```terminal
-npx @fixieai/fixie@latest serve
+$ fixie deploy
```
[the remark this answers is not in the document]
Comment (benlower) [reply packages/docs/docs/sidekicks-quickstart.md:119]: talked to Matt about this and we are aligned on this as best practice.

File: packages/docs/docs/sidekicks-quickstart.md [hunk 11]
```diff
@@ -27,212 +34,193 @@ At the end of this quickstart you will:
...
-🦊 Deploying agent sarah/sidekick-acme...
-👽 Updating agent sarah/sidekick-acme...
+$ fixie serve
 ```
 
-Once complete deployment completes, we would expect to see something like this:
+This starts up the Sidekick running on your local machine, and sets
+up a tunnel allowing the Fixie service to connect into your local
+Sidekick. When you quit the `fixie serve` command (for example, by
+hitting Ctrl-C), the Sidekick reverts back to the most recently
```
[the remark this answers is not in the document]
Comment (benlower) [reply packages/docs/docs/sidekicks-quickstart.md:160]: done

File: packages/docs/docs/sidekicks-quickstart.md [hunk 12]
```diff
@@ -27,212 +34,193 @@ At the end of this quickstart you will:
...
-_TODO Matt: need to figure out if we add this or not_
+> TODO(matt): Figure out what the right thing to do here is.
 
 <img src={FoxieSidekick} alt="Foxie, the Fixie mascot, doing a sidekick!" width="400"/>
 
 ## Additional Resources and Next Steps
 
 You've got a template Sidekick deployed to Fixie. So what's next? Here are some suggestions:
 
+> TODO(ben/matt): We should figure out if we have time to write more complete
```
[the remark this answers is not in the document]
Comment (benlower) [reply packages/docs/docs/sidekicks-quickstart.md:219]: Created https://github.com/fixie-ai/fixie/issues/1671 to track this work

**Comment by benlower** [comment]
i folded all these changes in via a diff PR

**Pull request closed without merging**